Dependencies/JOSESwift.xcframework/LICENSE

```diff
@@ -0,0 +1,176 @@
+Apache License
+Version 2.0, January 2004
+http://www.apache.org/licenses/
+
+TERMS AND CONDITIONS FOR USE, REPRODUCTION, AND DISTRIBUTION
+
```

Comment:
would it be good to add this as well? 
<img width="768" alt="Screenshot 2023-10-20 at 14 50 10" src="https://github.com/checkout/checkout-3ds-sdk-ios/assets/125963311/43b0cc96-719f-4cc0-8e4d-fdf697c6f7ef">

Reply:
Does Apache License tell us to do that?

Reply:
As far as I interpret it, it might be good to have this under the `Dependencies` folder. This should be the `NOTICE` that's mentioned in the license file